sopel/tools/jobs.py

```diff
@@ -179,6 +168,16 @@ def __init__(self, interval, func):
         self.interval = interval
         self.func = func
 
+    def is_ready_to_run(self, at_time):
+        """Check if this job is ready to run at the given time.
+
+        :param int at_time: Timestamp to check, in seconds
+        :return: ``True`` if the job is ready to run ``at_time`` time,
+                 ``False`` otherwise
```

Comment:
I'm not sure about the grammar of this sentence @dgw

Reply:
Hmm, naming the parameter `at_time` makes the grammatically perfect version sound awkward:

```
:return: ``True`` if the job is ready to run at ``at_time``,
         or ``False`` otherwise
```

The double-"at", :roll_eyes:

Also, "is" could become "is (or will be)", but that's getting awfully complicated again. That kind of thing should be reserved for the first line (would become "Check if this job is (or will be) ready to run at the given time.") instead of the return value line.